### ISSUE TITLE
Package maintainance

### ISSUE THREAD

**Issue by LecrisUT** [observing/pre-commit#165]
Worth pointing to the maintained alternatives:
- Fork of this repo: [fastify/pre-commit](https://github.com/fastify/pre-commit)
- [husky](https://typicode.github.io/husky/guide.html)

---

Hi there everyone,

I am looking forward to seeing more activity of this project being merged. Looking at all of the PRs, I see that there are quite a few nice PRs to have. I would like to get the authors together to organize in a fork to help modernize this package.

---

### Modernizations
- [ ] Github action
  - [ ] release workflow
  - [ ] CI testing
- [ ] Documentation
  - If you are new to npm, what are the step-by-step instructions to get this going
  - How to organize your pre-commits to keep the `package.json` file clean
  - How to integrate with your CI
  - What's the relation to pre-commit.com? How can we mimic that workflow
- [ ] Add pre-commit to this package and CI. Would be good to have popular pre-commits that are available on pre-commit.com 
- [ ] Update dependencies
- [ ] Merge PRs
  - [ ] #161 
  - [ ] #151
  - [ ] #164 
  - [ ] #97 or #75
  - [ ] #122?
- [ ] Address issues

In the meantime, is anyone willing to open up their fork to issues/discussions and provide a package that others can install? Looking at the forks @wheelerlaw seems to have some.

I am new to JS/TS so feel free to add to the list above with more suggestions

---

Pinging active forks and PR authors
@wheelerlaw, @jsumners, @verebes1, @nvasilakis, @jackmac92, @alexkuz, @kevinoid, @boneskull, @FabianMeul, @armata007, @JapSeyz, @observing, @3rd-Eden, @Swaagie

**Comment by boneskull** [comment]
I've moved on to husky, sorry

**Comment by LecrisUT** [comment]
Fair enough, I did also try tout husky and indeed works like a charm for anyone else looking at this.

**Comment by jsumners** [comment]
https://github.com/fastify/pre-commit

**Comment by wheelerlaw** [comment]
Let me refamiliarize myself with what I was working on here.
